src/main/res/values/strings.xml

```diff
@@ -18,7 +18,7 @@
     <string name="send_device_id_to">Send Device ID to</string>
 
     <!-- Text for FoldersFragment and DevicesFragment loading view -->
-    <string name="api_loading">Waiting for API</string>
+    <string name="api_loading">Loading...</string>
```

Comment:
Can you use an [Ellipsis](https://en.wikipedia.org/wiki/Ellipsis) here to fix that lint warning?